bot/admin/web/src/app/analytics/flow/flow.component.ts

```diff
@@ -160,7 +160,7 @@ export class FlowComponent implements OnInit, OnDestroy {
   flowData: ChartData;
   loading: boolean = false;
 
-  SankeyChartType: ChartType = ChartType.Sankey;
+  // SankeyChartType: ChartType = ChartType.Sankey;
```

Comment:
Can create an issue on tock repository to follow the change with  Apache Echarts  graphs and mention it (the issue number) as a comment here.

Reply:
https://github.com/theopenconversationkit/tock/issues/1714